src/app/external-cluster-wizard/steps/summary/external-cluster-details/provider/eks-cluster-summary/template.html

```diff
@@ -0,0 +1,54 @@
+<!--
+Copyright 2022 The Kubermatic Kubernetes Platform contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<km-external-cluster-details *ngIf="spec && cloudSpec"
+                             [cluster]="cluster">
+  <ng-container leftColumn>
+    <km-property>
+      <div key>Cluster Service Role</div>
+      <div value>
+        <span>{{ spec.roleArn }}</span>
+      </div>
+    </km-property>
+  </ng-container>
+
+  <ng-container rightColumn>
+    <div class="section-header">Networking</div>
+    <km-property>
+      <div key>VPC</div>
+      <div value>
+        <span>{{ spec.vpcConfigRequest.vpcId }}</span>
+      </div>
+    </km-property>
+
+    <km-property class="km-chips">
+      <div key>Subnet</div>
+      <mat-chip-list value>
```

Comment:
I think we can use `km-labels` here. Same for Security Groups. Wdyt?

Reply:
`km-labels` uses key-value pairs, these are only values.

Reply:
`km-labels` also works with array of strings.